docs/content/concepts/sui-move-concepts/packages/automated-address-management.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Automated Address Management
+---
+
+When you publish or upgrade a package, its address (also known as the package ID) is tracked in the `Move.lock` file. This bookkeeping is done automatically so that you can avoid recording or updating hex addresses (e.g., in the `Move.toml` file). 
+
+When you publish or upgrade your package on multiple chains (e.g., `mainnet`, `testnet`, `devnet`) separate addresses for each chain will be tracked for you. This tracking is based on your active environment (e.g., run `sui client active-env`). For example, if you set your active environment to an RPC connected to `testnet` and publish a package, the `Move.lock` will record the address for that package and associate it with your active environment (`testnet`).
```

Comment:
```suggestion
When you publish or upgrade your package on multiple chains (Mainnet, Testnet, Devnet) separate addresses for each chain are tracked for you. This tracking is based on your active environment (run `sui client active-env` if unsure). For example, if you set your active environment to an RPC connected to `testnet` and publish a package, the `Move.lock` records the address for that package and associates it with your active environment (`testnet`).
```
I think that first reference to networks should be regular but wouldnt hate it remaining lowercase in ticks.